website/content/docs/connect/config-entries/service-router.mdx

```diff
@@ -574,6 +574,69 @@ spec:
       description:
         'A list of HTTP response status codes that are eligible for retry.',
     },
+    {
+      yaml: false,
+      name: 'RequestHeaders',
+      type: 'HTTPHeaderModifiers: <optional>',
+      description: `A set of [HTTP-specific header modification rules](/docs/connect/config-entries/service-router#httpheadermodifiers)
```

Comment:
```suggestion
      description: `A set of [HTTP-specific header modification rules](#httpheadermodifiers)
```
minor suggestion: since this references an object on the same page, you can remove the URL path and just leave the fragment.

Reply:
I might leave this on the basis that this block is copy pasted between three config entries right now. If we remove the URL on this page it makes it harder to copy paste between them for future updates?